src/payment/strategies/klarna/klarna-payment-strategy.ts

```diff
@@ -121,6 +132,35 @@ export default class KlarnaPaymentStrategy implements PaymentStrategy {
             }));
     }
 
+    private _getUpdateSessionData(billingAddress: BillingAddress, shippingAddress?: Address): KlarnaUpdateSessionParams | {} {
+        if (billingAddress.countryCode === 'US') {
+            return {};
+        }
+
+        const data = {} as KlarnaUpdateSessionParams;
```

Comment:
Avoid using `as`. I'm guessing it was done this way because `shipping_address` is optional, you can just define `data` with the mandatory fields and append the optional ones if required.

```ts
const data: KlarnaUpdateSessionParams = {
    billing_address: this._mapToKlarnaAddress(billingAddress, billingAddress.email),
};

if (shippingAddress) {
    data.shipping_address = this._mapToKlarnaAddress(shippingAddress, billingAddress.email);
}

return data;
```